pkg/collector/corechecks/oracle-dbm/config/config.go

```diff
@@ -136,6 +136,11 @@ Port: '%d'
 `, c.Server, c.ServiceName, c.Port)
 }
 
+// GetDefaultObfuscatorOptions return default obfuscator options
+func GetDefaultObfuscatorOptions() obfuscate.SQLConfig {
+	return obfuscate.SQLConfig{DBMS: common.IntegrationName, TableNames: true, CollectCommands: true, CollectComments: true, ObfuscationMode: obfuscate.ObfuscateAndNormalize}
```

Comment:
Do you plan to expose a config option for `ObfuscationMode` so that customer can turn it off if needed?

Reply:
FIrst I wanted to, but then realized that there were many bugs in the old obfuscator. Fixing bugs as they come might be a better option.